api/envoy/type/tracing.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+package envoy.type;
+
+option java_outer_classname = "TracingProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.type";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Tracing]
+
+// Describes custom tags for the active span.
+message TracingCustomTag {
+  // Literal type custom tag with literal value for tag value.
+  message Literal {
+    // literal value to populate the tag value.
```

Comment:
nit: Please use proper grammar for all doc comments including starting sentence with a capital letter. Same elsewhere.

Reply:
fixed.